Cargo.toml

```diff
@@ -65,6 +65,7 @@ syn2 = { package = "syn", version = "2.0.38", default-features = false }
 quote = "1.0.33"
 manyhow = { version = "0.8.1", features = ["darling"] }
 darling = "0.20.3"
+drop_bomb = "0.1.5"
```

Comment:
this is so cool

Reply:
It would be great to have linear types in rust to check this statically, but runtime checking is still better than nothing